kubeflow/examples/prototypes/tf-job-simple.jsonnet

```diff
@@ -10,7 +10,7 @@ local namespace = "default";
 local image = "gcr.io/kubeflow/tf-benchmarks-cpu:v20171202-bdab599-dirty-284af3";
 
 local tfjob = {
-  apiVersion: "kubeflow.org/v1alpha1",
+  apiVersion: "kubeflow.org/v1alpha2",
```

Comment:
Why did you change this? This is not correct. The other changes needed to make the spec v1alpha2. Can we just leave tf-job-simple as v1alpha1 for now.

Reply:
Sure, let's merge this one first.